app/components/ui/molecules/PeoplePicker.js

```diff
@@ -209,13 +170,14 @@ class PeoplePicker extends React.Component {
     let extendedPeople = [...people].sort((a, b) =>
       a.name.localeCompare(b.name),
     )
-    // TODO add person.aff once we have that info
+
     extendedPeople = extendedPeople.map(person => ({
       ...person,
-      searchValue: `${person.name} ${person.subjectAreas.join(' ')}`,
+      searchValue: `${person.name} ${person.subjectAreas.join(' ')} ${
+        person.aff ? person.aff : ''
```

Comment:
Not entirely sure now that there is only one affiliation for each person? It's called `affiliations` after all.

Reply:
Good point - but we can fix that later

Reply:
Just like we could have added `person.aff` later once we have it (which is what I would've done) but this is fine as well :)